starter-code/basic-algorithms.js

```diff
@@ -1,7 +1,32 @@
 // Names and Input
+let hacker1 = prompt('what\'s the driver\'s name?');
+    console.log('The driver\'s name is ' + hacker1);
 
+let hacker2 = prompt('what\'s the navigator\'s name?');
+    console.log('The navigators\'s name is ' + hacker2);
 
 //Conditionals
 
+if (hacker1.length > hacker2.length) {
+        console.log("The Driver has the longest name, it has XX characters");}
+
+    else if (hacker1.length < hacker2.length) {
+        console.log("Yo, navigator got the longest name, it has XX characters");}
```

Comment:
Para que nos muestre cuantos caracteres tiene podríamos concatenar hacker1.length "The Driver has the longest name, it has " +  hacker1.length + "characters", asi la consola nos mostrara el numero de caracteres que contiene el nombre.